content/en/tracing/trace_collection/custom_instrumentation/java/otel.md

```diff
@@ -181,6 +181,55 @@ public class Example {
 }
 ```
 
+## Adding span events
+
+_Minimum SDK version: 1.40.0._
```

Comment:
```suggestion
<div class="alert alert-info">Adding span events requires SDK version 1.40.0 or higher</div>
```

Reply:
Than you! Added and updated related docs in https://github.com/DataDog/documentation/pull/25157/commits/9e2c256728da10da5a7a200c66fbcd8463d9c43a.